docs/console/company-configuration/providers/extensions/orchestrator-generator.md

```diff
@@ -0,0 +1,245 @@
+---
+id: orchestrator-generator
+title: Orchestrator Generator
+sidebar_label: Orchestrator Generator
+---
+
+Capability: `Orchestrator Generator`
+
+An Orchestrator Generator can generate orchestrator's **files**, ready to **deploy**, which are saved in the **configuration repository**. For instance, it could generate Kubernetes files.
```

Comment:
```suggestion
An Orchestrator Generator can generate orchestrator's **files**, ready to be **deployed**; these files are saved in the **configuration repository**. For instance, it could generate Kubernetes files.
```